src/redux/configStore.js

```diff
@@ -0,0 +1,14 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import detailsReducer from './details/details';
+import homeReducer from './home/home';
+
+const rootReducer = combineReducers({
+  homeData: homeReducer,
+  detailsData: detailsReducer,
+});
+
+const store = createStore(rootReducer, applyMiddleware(logger, thunk));
+
```

Comment:
- [ ] Please Do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.